test/k8sT/manifests/graceful-termination.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: graceful-term-svc
+spec:
+  ports:
+    - port: 8081
+  selector:
+    app: graceful-term-server
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: graceful-term-server
+  labels:
+    app: graceful-term-server
+spec:
+  containers:
+    - name: server
+      image: docker.io/cilium/graceful-termination-test-apps:latest
+      imagePullPolicy: IfNotPresent
+      ports:
+        - containerPort: 8081
+          protocol: TCP
+      command: [ "/server", "8081" ]
+  terminationGracePeriodSeconds: 15
+  tolerations:
+    - effect: NoSchedule
+      key: node-role.kubernetes.io/master
+    - effect: NoSchedule
+      key: node.cloudprovider.kubernetes.io/uninitialized
+      value: "true"
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: graceful-term-client
+  labels:
+    app: graceful-term-client
+spec:
+  restartPolicy: OnFailure
+  terminationGracePeriodSeconds: 0
+  containers:
+    - name: client
+      image: docker.io/cilium/graceful-termination-test-apps:latest
+      imagePullPolicy: IfNotPresent
+      command: [ "/client", "graceful-term-svc.default.svc.cluster.local.:8081" ]
+---
+apiVersion: apps/v1
```

Comment:
This is a duplicate of the `test/k8sT/manifests/demo_ds.yaml`.

Reply:
No, it's not. It contains a bunch of other test apps. :( See my previous comment - https://github.com/cilium/cilium/pull/17716#discussion_r745769554.

Reply:
I meant this particular object is defined in both files.